base-node/index.d.ts

```diff
@@ -131,6 +139,23 @@ export interface NodeOptions<Headers extends object = {}> {
    */
   inMap?: LogMapper
 
+  /**
+   * Function that will be called instead of `inMap`, `inFilter` and `Log#add`.
+   *
+   * Use it if you want more control over when an action's `access` callback will be called
+   * and when an action will be added to the log.
+   *
+   * ```js
+   * onSync(processAction, action, meta) {
+   *   // Process an action later
+   *   myActionQueue.schedule(async () => {
+   *     await processAction(action, meta) // will call `inMap`, `inFilter` and `Log#add`
+   *   })
+   * }
+   * ```
+   */
+  onSync?: SyncCallback
```

Comment:
I have plan later to replace `inFiler`/`inMap` with `onSync` (are you agree that it is possibe?)

Can you rename `onSync` to `onReceive` (so later we will be able to add `onSend` for `outFilter`/`outMap`)?

Reply:
Yes this should be possible